app/uk/gov/hmrc/gform/gform/EmailAuthController.scala

```diff
@@ -62,6 +63,8 @@ class EmailAuthController(
 
   private val logger: Logger = LoggerFactory.getLogger(getClass)
 
+  private val emailConfirmationCode = "GEAD"
```

Comment:
Probably better to default to ABCD instead, easy to remember

Reply:
And also use `ci` right here ie `ci"ABCD"` to avoid that interpolation gymnastics down below: `ci"$emailConfirmationCode"`.